events/app.py

```diff
@@ -2,7 +2,7 @@
 import datetime
 import json
 import pymongo
-from bson import json_util
+from bson import json_util, ObjectId
```

Comment:
Where is `ObjectId` used?

Reply:
It's for serializing the `_id` field when searching for events by ID. MongoDB doesn't let you search for a field of type ObjectId by string, so you have to create an ObjectId yourself. See line 67.

Reply:
D'oh... I totally missed this while scanning.

Thanks!